src/main/target/FLYWOOF405S_AIO/target.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of INAV.
+ *
+ * INAV is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * INAV is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with INAV.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <platform.h>
+#include "drivers/io.h"
+#include "drivers/pwm_mapping.h"
+#include "drivers/timer.h"
+
+timerHardware_t timerHardware[] = {
+    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MC_MOTOR  | TIM_USE_FW_SERVO,   0, 1), // S1
```

Comment:
Can you change the TIM_USE_MC_MOTOR | TIM_USE_FW lines to TIM_USE_OUTPUT_AUTO?

Reply:
TIM_USE_OUTPUT_AUTO and TIM_USE_MOTOR are both causing compile errors in 6.1.1 and 7.0.0. In which files are these defined?

Reply:
If you are having issues with 7, you may need to pull the latest changes into your repo.

Reply:
Thank you for explaining. Done.